### PR TITLE
Generate tokens for sa/image-puller on app.ci

### PR DESCRIPTION
**Pull request by hongkailiu**
https://issues.redhat.com/browse/DPTP-1500

About the commit: Generate tokens for `sa/image-puller on app.ci`:

This token will be used e.g., in https://github.com/openshift/release/blob/master/clusters/build-clusters/01_cluster/openshift/ci-operator/registry-pull-credentials_secret_template.yaml

We do not need this now but we are going to need it after migrating the registry.
https://issues.redhat.com/browse/DPTP-1655
Since we are handling the SA name used in the secret, we can do it together.

/cc @alvaroaleman @stevekuznetsov

### CODE REVIEW

**Comment by hongkailiu** [comment]
To Delete:

BitWarden
Item: quay.io
Field: pull-token

```
oc --context app.ci delete secret -n ci quay-io-pull-credentials

oc --context api.ci delete sa -n ci build01
oc --context app.ci delete sa -n ci build01
oc --context build01 delete sa -n ci build01
oc --context build02 delete sa -n ci build01

oc --context api.ci delete clusterrolebinding image-puller-build01
oc --context app.ci delete clusterrolebinding image-puller-build01
oc --context build01 delete clusterrolebinding image-puller-build01
oc --context build02 delete clusterrolebinding image-puller-build01
```

Clean up the mounted secrets in `applyconfig` jobs.

**Comment by hongkailiu** [comment]
With current issues ongoing on build farm ...
Let us hold this for a bit

/hold

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 1]
```diff
@@ -1756,3 +1744,143 @@ secret_configs:
...
+  #    from the release payload and they are in quay.io/openshift-release-dev/...
+  - from:
+      .dockerconfigjson:
+        dockerconfigJSON:
+        - bw_item: quay.io
+          registry_url: quay.io
+          auth_bw_attachment: pull-token
+        - bw_item: build_farm
+          registry_url: docker-registry.default.svc:5000
+          auth_bw_attachment: token_image-puller_ci_reg_auth_value.txt
```
Comment (stevekuznetsov) [inline core-services/ci-secret-bootstrap/_config.yaml:1766]: nit: all of these are `token_<sa>_<cluster>_auth_value.txt` but for `api.ci` we just have `ci` for the cluster name - why?

Comment (hongkailiu) [reply core-services/ci-secret-bootstrap/_config.yaml:1766]: Historically, it is called that way.
Like my other bad naming conventions.
It is on the BW for existing BW items.

They will be removed after api.ci is gone.

**Comment by hongkailiu** [comment]
I will merge it when build farm is more stable with 4.6.

**Comment by hongkailiu** [comment]
Deleted those on BW:

```
ll bk.build01_reg_auth_value
total 32
-rw-r--r--@ 1 hongkliu  staff   1.2K  2 Nov 08:34 build01_build01_reg_auth_value.txt
-rw-r--r--@ 1 hongkliu  staff   1.2K  2 Nov 08:34 build01_build02_reg_auth_value.txt
-rw-r--r--@ 1 hongkliu  staff   1.1K  2 Nov 08:34 build01_ci_reg_auth_value.txt
-rw-r--r--@ 1 hongkliu  staff   1.2K  2 Nov 08:34 build01_vsphere_reg_auth_value.txt
```

**Pull request merged**